docs/v3/core/v3.0.rst

```diff
@@ -1273,6 +1271,8 @@ when opening a Zarr hierarchy to automatically determine the appropriate store.
 Abstract store interface
 ------------------------
 
+(This subsection is not normative.)
```

Comment:
I feel like this could be a lot clearer, e.g:

```suggestion
(This subsection is not normative. It is intended only as a recommendation for API design, but does not need to be strictly followed like the specification for the on-disk storage does.)
```